### ISSUE TITLE
import csv file issue

### ISSUE THREAD

**Issue by ignacio0821** [Show-Me-the-Code/python#432]
Hello I am stuck on a code to iterate from a .csv file and output the month and year. I am only outputting the Month at the moment. Can I see some examples of what I am doing wrong or the correct functions for this?

Thanks,

**Comment by ignacio0821** [comment]
Yes, would you like to see the def process_row code I am working on?'

Thanks,

**Comment by ankit-gautam23** [comment]
Hey man, I would love to solve this issue, assign me if not assigned to anyone else.

**Comment by shlomo-mica** [comment]
Hi can  i try ? thanks
